capactio/capact
# Increase max_connections for postgresql

**Description**

Changes proposed in this pull request:

- Set `max_connections` to 1000

**Testing**

1. Deploy CloudSQL using Terraform runner.
2. Login into DB: `psql -h <IP> -U <username> <db>`
3. Verify that `max_connections` is set to 1000
    ```sql
    show max_connections;
    ```


**Related issue(s)**
